lib/TokenManager.js

```diff
@@ -180,11 +197,7 @@ function clear(tokenMgmtRef, storage) {
 }
 
 function TokenManager(sdk, options) {
-  options = options || {};
-  options.storage = options.storage || 'localStorage';
-  if (!options.autoRenew && options.autoRenew !== false) {
-    options.autoRenew = true;
-  }
+  options = Object.assign({}, DEFAULT_OPTIONS, util.removeNils(options));
```

Comment:
Is our webpack build going to convert `Object.assign` properly for IE support?

Reply:
Good catch, back to `util.extend` !